### PR TITLE
Upgrade ngx_small_light

### PR DESCRIPTION
**Pull request by agatan**
The latest version of ngx_small_light is 0.9.1. https://github.com/cubicdaiya/ngx_small_light/blob/master/ChangeLog

v0.8.0 contains some bugs and one of those bugs is critical for me (already fixed in the latest version).

### CODE REVIEW

**Comment by creasty** [comment]
Kawaisou

File: Dockerfile [hunk 1]
```diff
@@ -22,6 +22,7 @@ RUN apt-get update && \
 # Build ImageMagick with WebP support
 RUN mkdir -p /tmp/imagemagick && \
     cd /tmp/imagemagick && \
+    echo "\ndeb-src http://archive.ubuntu.com/ubuntu/ trusty main restricted" >> /etc/apt/sources.list && \
```
Comment (bgpat) [inline Dockerfile]: Why do you add this line?

Comment (agatan) [reply Dockerfile]: `apt-get build-dep` すると

```
Reading package lists... Done
E: You must put some 'source' URIs in your sources.list
```

で失敗したので足しました。

**Review by koudaiii — APPROVED** [review]
LGTM 👀

**Pull request merged**